components/BackBtn.js

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
+import styles from "./BackBtn.module.css";
+
+const BackBtn = () => {
+  const router = useRouter();
+  return (
+    <>
+      <button className={styles.container} onClick={() => router.back()}>
```

Comment:
There's always a risk involved in using this type of "back" navigation, because if the user found the page via Google or some other external link, clicking "back" will take them back to wherever they came from, i.e. leave the site.